test/EnvironmentManagerTests/EnvironmentManagerWin32Tests.h

```diff
@@ -11,5 +11,8 @@ namespace ProjectReunionEnvironmentManagerTests
         TEST_METHOD(TestGetForProcess);
         TEST_METHOD(TestGetForUser);
         TEST_METHOD(TestGetForMachine);
+        TEST_METHOD(TestGetEnvironmentVariablesForProcess);
+        TEST_METHOD(TestGetEnvironmentVariablesForUser);
+        TEST_METHOD(TestGetEnvironmentVariablesForMachine);
```

Comment:
Where's all the fun RunAs permutations

RunAs:RestrictedUser = MediumIL
RunAs:ElevatedUserOrSystem = Elevated

Dupe all these to cover both those RunAs cases?

Reply:
Something like this?